### PR TITLE
Prettify Identifier

### PR DESCRIPTION
**Pull request by camjac251**
Similarly to Genius over at https://github.com/khanhas/genius-spicetify/blob/master/manifest.json

This would allow the theme Dribbblish to show the title of the app neatly.

### CODE REVIEW

**Comment by Kasama** [comment]
Hey @camjac251! First of all, thanks for the contribuition.

It was, in fact, like this [in the past](https://github.com/Kasama/spicetify-jqbx/blob/b2d69e455f41be575b15a71c8ba3cc57b5a3293e/manifest.json#L2), but i changed it to match the name of the repository.

 The issue is that if a CustomApp has a `BundleIdentifier` that's different from the directory the app is in, the app won't load in spotify.

Genius app works around that by adding a `mv` command to the installation guide where it renames the directory to match the manifest.

I think it's a good idea to change it back to just `JQBX`. Would you mind also updating the readme to instruct people installing this Custom App to rename its directory to just `JQBX`?

**Comment by camjac251** [comment]
I've updated the instructions on the readme to reflect it.

Although it might be worth to also add the scoop installation method instructions which I use for spicetify. 
https://github.com/lukesampson/scoop/wiki/So-What
https://github.com/TheRandomLabs/Scoop-Spotify

This I find very useful as it would allow for updating with a single `scoop update *` when repositories get updates. Looks like the themes repo just was yesterday.

File: README.md [hunk 1]
```diff
@@ -9,15 +9,16 @@ How to install
...
 
   Linux: `$XDG_CONFIG_HOME/.config/spicetify/CustomApps/` or `~/.config/spicetify/CustomApps`
 
   MacOS: `~/spicetify_data/CustomApps`
 
-3. Add the app to your spotify by running `spicetify config custom_apps spicetify-jqbx` or change your `config.ini` to include the `spicetify-jqbx` app
+3. Add the app to your spotify by running `spicetify config custom_apps JQBX` or change your `config.ini` to include the `JQBX` app folder.
 
 ```ini
 [AdditionalOptions]
```
Comment (Kasama) [inline README.md:24]: I think you only forgot to change the `custom_apps` name in the `[AdditionalOptions]`

**Comment by Kasama** [comment]
Nice! I didn't know about Scoop.

Please feel free to add a section to the readme saying how to install this app using it.

File: README.md (hunk 1 above)
Comment (camjac251) [reply README.md:24]: I always seem to forget something...

Should be fixed now

**Comment by camjac251** [comment]
Is this ready to be merged? I'd add scoop later

**Review by Kasama — APPROVED** [review]

**Comment by Kasama** [comment]
Thanks @camjac251!

You are welcome to also add that documentation about Scoop if you feel like it :smiley:

**Pull request merged**